scipy/_lib/_array_api.py

```diff
@@ -126,7 +126,10 @@ def array_namespace(*arrays: Array) -> ModuleType:
 
     _arrays = compliance_scipy(_arrays)
 
-    return array_api_compat.array_namespace(*_arrays)
+    try: # return the wrapped namespace for NumPy arrays
+        return array_api_compat.array_namespace(*_arrays, use_compat=True)
+    except ValueError: # if the library is not wrapped, like array-api-strict
+        return array_api_compat.array_namespace(*_arrays, use_compat=None)
```

Comment:
https://github.com/data-apis/array-api-compat/issues/168

Reply:
If we want to merge this PR before resolving this, it'd be good to add a code comment about `try:` to refer to the upstream issue.

Reply:
done!

Reply:
upstream PR is in, so this can be removed before merge

Reply:
Want to make the change now? I'm also fine with doing it as a follow-up if you prefer.